NickvisionTubeConverter.GNOME/Views/KeyringDialog.cs

```diff
@@ -330,4 +343,42 @@ private void DeleteCredential(Gtk.Button sender, EventArgs e)
         };
         disableDialog.Present();
     }
+
+    /// <summary>
+    /// Occurs when the enable button is clicked under a snap
+    /// </summary>
+    /// <returns>True if slot is connected and False if not</returns>
+    public async Task<bool> SnapSlotCheckAsync()
+    {
+        var process = new Process();
+        process.StartInfo.FileName = "snapctl";
+        process.StartInfo.Arguments = "is-connected password-manager-service";
+        process.StartInfo.CreateNoWindow = true;
+        process.Start();
+        await process.WaitForExitAsync();
+        int connected = process.ExitCode;
+        if (connected == 1)
+        {
+            var snapDialog = Adw.MessageDialog.New(this, _("Necessary slot not connected"), _("To enable keyring, the app needs to connect to the password-manager-service slot. To connect it please copy the command and run in the terminal."));
```

Comment:
Title: `Snap Slot Not Connected`
Description: `To enable keyring, the app needs to connect to the password-manager-service snap slot. To connect to it, please copy the command and run it in the terminal.`

Reply:
Not sure if the title `Snap slot not connected` , coz it might mislead people to think that they connected a slot named snap.

Reply:
I'd just make the title `Missing Slot` in that case